presto-docs/src/main/sphinx/sql/call.rst

```diff
@@ -12,7 +12,7 @@ Synopsis
 Description
 -----------
 
-Call a procedure.
+Call a procedure defined for a given connector.
```

Comment:
How about "Call a connector-defined procedure."

Though the fact that a procedure is connector-defined (or comes from a connector) seems like an implementation detail that shouldn't be part of the SQL language documentation.